bin/rose

```diff
@@ -176,6 +239,14 @@ doc)
     :;;
 esac
 
+CMD="${ROSE_NS}-${UTIL}"
```

Comment:
This works fine for me if I do `rose suite-run`, but if I do `rose`/`rose --help` the help message contains the old info: i.e.
```
    suite-run
        Install and run a Cylc suite.
```

Reply:
I think you have probably got some old Rose files kicking about in your environment, steps to remove them here: https://github.com/metomi/rose/pull/2446#issuecomment-800342335

Reply:
No, that doesn't seem to fix it. It works for you?

Reply:
It is definitely not coming from this branch because:

```console
$ git grep 'Install and run a Cylc suite'
```

Could your wrapper script be picking up Rose2019?

Reply:
OK now - I copied your rather more sophisticated wrapper script.